source/includes/_api.md

```diff
@@ -213,13 +214,90 @@ $ curl -XGET http://localhost/api/v2/events?_size=4&_offset=1&_include=@timestam
 }
 ```
 
-You can receive a subset of your query by using two parameters:
+If the response includes too many items for the client to handle at once, use pagination to get only a subset of the
+results, defined by two parameters:
 
-* `_size` (default: 10000) the max size of the result subset you'd receive.
+* `_size` (default: 10000) the max size of the result subset to receive.
 * `_offset` (default: 0) the number of resources to skip, i.e. `_offset=1` means you skip the first resource.
 
 \* both parameters are optional.
 
-The response metadata returns your requested parameters, and a `total` field which indicates the size of the full set.
+The response metadata returns the requested parameters, and a `total` field which indicates the size of the full set.
 
 ## Authentication
+
+Authentication headers should be added to each request sent to a secured manager.<br>
+Any header can be used, as long as it's support by the manager's authentication provider, which eventually process it.<br>
```

Comment:
which eventually process it -> which will eventually process it (or which eventually processes it)